cmd/mimir/config-descriptor.json

```
@@ -3223,6 +3223,17 @@
               "fieldFlag": "ingester.circuit-breaker.push-timeout",
               "fieldType": "duration",
               "fieldCategory": "experiment"
+            },
+            {
+              "kind": "field",
+              "name": "read_timeout",
+              "required": false,
+              "desc": "How long is execution of ingester's read-path request supposed to last before it is reported as timeout in a circuit breaker. This configuration is used for circuit breakers only, and timeout expirations are not reported as errors",
```

I've made an attempt to simplify the language and be more clear.

```suggestion
              "desc": "The maximum length of time an ingester's read-path request can last before it triggers a circuit breaker. This configuration is used for circuit breakers only, and its timeouts aren't reported as errors",
```

---

Thank you